deps.bzl

```diff
@@ -30,6 +30,14 @@ def shellcheck_dependencies():
         "linux_arm64": "179c579ef3481317d130adebede74a34dbbc2df961a70916dd4039ebf0735fae",
     }
 
+    # pull darwin_aarch64 binaries from vscode-shellcheck/shellcheck-binaries
```

Comment:
Interesting, I see that the shellcheck from VSCode binaries comes as a tar.gz whereas shellcheck from the official website is a tar.xz archive. I wonder if there is a way to unify this in some way. My initial idea was that it would be great to have a second URL for each platform that is pointing to the VSCdoe URLs.